code/datums/elements/chemical_transfer.dm

```diff
@@ -0,0 +1,59 @@
+/**
+ * ## chemical transfer element!
+ *
+ * Bespoke element that, on a certain chance to proc, transfers all chemicals from the person attacking to the victim being attacked
+ * with whatever item has this element.
+ *
+ * attacker_message uses %VICTIM as whomever is getting attacked.
+ * victim_message uses %ATTACKER for the same.
+ */
+/datum/element/chemical_transfer
+	element_flags = ELEMENT_BESPOKE|ELEMENT_DETACH
+	id_arg_index = 2
+	///chance for the chemical transfer to proc.
+	var/transfer_prob
+	///message attacker gets when the chemical transfer procs
+	var/attacker_message
+	///message victim gets when the chemical transfer procs
+	var/victim_message
+
+/datum/element/chemical_transfer/Attach(datum/target, attacker_message = span_notice("You transfer your chemicals to %VICTIM."), victim_message = span_userdanger("Chemicals have been transferred into you from %ATTACKER!"), transfer_prob = 100)
```

Comment:
Make these text inserts defines. Wanna reduce the chance of fuckup as much as we can.

Reply:
👍